Mathlib/Algebra/GradedMonoid.lean

```diff
@@ -649,27 +649,29 @@ section HomogeneousElements
 variable {R S : Type*} [SetLike S R]
 
 /-- An element `a : R` is said to be homogeneous if there is some `i : ι` such that `a ∈ A i`. -/
-def SetLike.Homogeneous (A : ι → S) (a : R) : Prop :=
+def SetLike.IsHomogeneousElem (A : ι → S) (a : R) : Prop :=
```

Comment:
It is not possible to add deprecation aliases because something else took the name, but can you add the aliases for the rest of the theorems (if applicable) and document this rename in the PR message? Thanks!

Reply:
done